extension/src/lib/highContrast.js

```diff
@@ -0,0 +1,87 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+// This component makes use of css/highContrast.css to perform the adaptations
+// of the web content, and this is done through chrome.tabs.executeScript.
+//
+fluid.defaults("gpii.chrome.highContrast", {
+    gradeNames: ["fluid.modelComponent"],
+    scriptTemplate: "document.documentElement.setAttribute('data-gpii-hc','__THEME__');" +
```

Comment:
Crikey, what is this! Don't we believe in putting scripts into files any more? : P